Adyen/Utilities/Localization.swift

```diff
@@ -159,3 +154,18 @@ private func localizedZeroPaymentAuthorisationButtonTitle(
         return localizedString(.confirmPreauthorization, parameters)
     }
 }
+
+extension [LocalizationInput] {
+
+    fileprivate mutating func buildPossibleInputs(
```

Comment:
maybe 
```suggestion
    fileprivate mutating func appendPossibleInputs(
```

Would help to differentiate from

```
private func buildPossibleInputs(
    _ key: String,
    _ parameters: LocalizationParameters?
) -> [LocalizationInput] {}
```

Reply:
Good point :)

Reply:
Fixed here [ff679ab](https://github.com/Adyen/adyen-ios/pull/2108/commits/ff679ab89496c06f6a09583bd9c58278d1391962)